proxy/src/auth/backend/link.rs

```diff
@@ -79,15 +79,19 @@ pub(super) async fn authenticate(
 
     // Give user a URL to spawn a new database.
     info!(parent: &span, "sending the auth URL to the user");
+    let pause = ctx.latency_timer.pause(crate::metrics::Waiting::Client);
     client
         .write_message_noflush(&Be::AuthenticationOk)?
         .write_message_noflush(&Be::CLIENT_ENCODING)?
         .write_message(&Be::NoticeResponse(&greeting))
         .await?;
+    drop(pause);
 
     // Wait for web console response (see `mgmt`).
     info!(parent: &span, "waiting for console's reply...");
+    let pause = ctx.latency_timer.pause(crate::metrics::Waiting::Cplane);
```

Comment:
I guess, it's not fully true, that it's Cplane, rather, it's client clicking on the web interface. Probably we could have a single pause.

Reply:
Maybe. I don't think we should worry too much about this though